### ISSUE TITLE
Can not run on macosx

### ISSUE THREAD

**Issue by Andrewthedev** [ExposureSoftware/TEC-Client#83]
I have tried installing the modules using pip and everytime i fix one module another one breaks. Can you put up the system requirements for macosx or linux in order to run this program from the command line?

**Comment by marshall-davis** [comment]
Sorry it took so long to reply.

A fix for OSX was added at some point, and it involved an icon that was Windows specific. Doesn't sound like that's your issue. The `requirements.txt` file in the repo looks correct to me but I'll be sure and check it tonight. It may take more than one night though, I have to borrow an OSX machine for testing.

**Comment by Andrewthedev** [comment]
No worries bro. I got it working after all. I had to install some modules
and   delete some of the code. Like the logging wasn't the same on osx as
Windows. How do I create my own addons like Patrick did?

**Comment by marshall-davis** [comment]
Could you provide the output of `pip freeze` on that setup so I can see what needs to be changed on OS X?  
The plug-ins are written in Python and the script must be stored in the plugins directory. Where this is located may be different on OS X. There's a few events that can trigger and action from plugin, and right now you'd have to look at the code to find them. I'll try and document that.  
Personally, I wasn't a huge fan of the idea because it opens the doors for this to be the "cheating" client so I didn't pay much attention to them other than "did not break the client".

**Comment by Andrewthedev** [comment]
Yeah, I can get on my mac this weekend and let you know the steps I took in order to run the application. I had to run it through command line in order for it to work. I am now running it on a windows machine and the differences I see initially are there is actually no menu bar on the mac version for some reason. Also, I put the files from patrick's github in the plugins directory and none of them came up in the list so i'm not sure how that works either.

**Comment by marshall-davis** [comment]
Like I said, I can document that for you. No menu bar on the Mac is normal
I think. I think you must have the client focuses and it should show up in
the very top menu of OS X. If not, we'll have to look into it.

**Comment by Andrewthedev** [comment]
Here is the pip3 freeze 

appdirs==1.4.0
cx-Freeze==4.3.3
inflect==0.2.5
jaraco.apt==1.0
jaraco.classes==1.2
jaraco.collections==1.1
jaraco.context==1.3
jaraco.functools==1.2
jaraco.itertools==1.3
jaraco.logging==1.1
jaraco.text==1.3
more-itertools==2.2
nose==1.3.6
requests==2.6.2
six==1.9.0
tempora==1.2
virtualenv==12.1.1
yg.lockfile==2.0
zc.lockfile==1.1.0

I did remove some lines of code for it to work. Also, I'm running it using python3.5

**Comment by marshall-davis** [comment]
It was built on Stackless Python 3.3, but that shouldn't make a difference.
